### ISSUE TITLE
Types in this library stutter

### ISSUE THREAD

**Issue by tcharding** [bitcoindevkit/rust-hwi#81]
The types in this library use a prefix `HWI` which makes them stutter if used with the full path (eg. `hwi::HWIAddress`). We could simplify the path and make the code more terse by removing all the prefixes. 

(Please note, I have not spent much time in this repo so I may be missing some nuance.)

**Comment by danielabrozzoni** [comment]
I agree, removing the HWI prefix is I think the right way forward :)
